app/assets/stylesheets/common/base/topic-post.scss

```diff
@@ -809,10 +809,11 @@ aside.quote {
         align-items: center;
         justify-content: center;
         box-sizing: border-box;
-        border-radius: 0 0.25em 0.25em 0;
+        border-radius: 0 var(--border-radius) var(--border-radius) 0;
         padding-top: 0;
         padding-bottom: 0;
-        height: 100%;
+        height: calc(100% + 0.25em);
```

Comment:
Is the extra `0.25em` here necessary? Feels odd, given that this removes the same value from the border-radius (and replaces it with `--border-radius`, which in core is set at 2em).

Reply:
I shouldn't have used 0.25em, but a flat 4px.
It allows the X button to be as tall as the parent, which has a vertical padding of 4px (2 top +2 bottom).
```css
.archetype-private_message .topic-map .participants .user {
    border: none;
    background: var(--primary-low);
    padding: 2px 4px; /* <------- */
    border-radius: var(--border-radius);
}
```
I don't see a way to keep just 100% height on the X button as long as the parent has vertical padding… But I may ignore some CSS tricks :)